site/docs/recipes/relations.md

```diff
@@ -7,13 +7,13 @@ Kysely IS a query builder. Kysely DOES build the SQL you tell it to, nothing mor
 
 Phew, glad we got that out the way..
 
-All that was said above doesn't mean there's no way to nest related rows in your queries. 
-You just have to do it with the tools SQL and the underlying dialect (e.g. PostgreSQL or MySQL) provide.
-In this recipe we show one way to do that when using the built-in PostgreSQL and MySQL dialects.
+All that was said above doesn't mean there's no way to nest related rows in your queries.
+You just have to do it with the tools SQL and the underlying dialect (e.g. PostgreSQL, MySQL, or SQLite) provide.
+In this recipe we show one way to do that when using the built-in PostgreSQL, MySQL, and SQLite dialects.
 
 ## The `json` data type and functions
 
-Both PostgreSQL and MySQL have rich JSON support through their `json` data types and functions. `pg` and `mysql2`, the node drivers, automatically parse returned `json` columns as json objects. With the combination of these two things, we can write some super efficient queries with nested relations.
+PostgreSQL, MySQL, and SQLite have rich JSON support through their `json` data types and functions. `pg` and `mysql2`, the node drivers, automatically parse returned `json` columns as json objects. With the combination of these two things, we can write some super efficient queries with nested relations.
```

Comment:
Let's suggest that in SQLite's case, you'd need [ParseJSONResultsPlugin](https://github.com/kysely-org/kysely/blob/master/src/plugin/parse-json-results/parse-json-results-plugin.ts) for automatic parsing since the driver doesn't do it?

Reply:
@igalklebanov mentioned the `ParseJSONResultsPlugin` in the docs with example usage 👍